lib/vagrant/util/mime.rb

```diff
@@ -0,0 +1,86 @@
+require 'mime/types'
+require 'securerandom'
+
+module Vagrant
+  module Util
+    module Mime
+      class Multipart
+
+        # [List] collection of content part of the multipart mime
```

Comment:
Docs here should list it as a `@return` and the `[List]` should be an `Array`, ideally of the type inside:

```ruby
# @return [Array<String>] collection of ...
```